interactive_engine/compiler/conf/ir.compiler.properties

```diff
@@ -61,3 +61,15 @@ calcite.default.charset: UTF-8
 
 # set the max capacity of the result streaming buffer for each query
 # per.query.stream.buffer.max.capacity: 256
+
+# set the mode to read ir meta, either from a 'local' file or via remote 'http'.
+# ir.meta.reader.mode: local
```

Comment:
The configuration of ir.meta.xx, should be uniformly defined with graph.schema and graph.statistics. In addition, we can really configure those service scheme, to automatically differentiate from local and http. For example, 

graph.meta.statistics.uri: http://localhost:8080  -> this is a http file, or 
graph.meta.statistics.uri: file:///path/to/local/file -> this is a local file

Also give a uri for schema

Reply:
done